rmd_files/RNAseq_report.Rmd

```diff
@@ -2155,10 +2153,21 @@ if ( dataset == "PAAD" || dataset == "PAAD-IPMN" || dataset == "PAAD-NET" || dat
   comp_cancer_group <- ext_cancer_group
 }
 
+##### Read in Dragen's mapping metrics file (if exists) to extract number of reads
+dragen_mapping_metrics <- paste(params$dragen_rnaseq, list.files(params$dragen_rnaseq, pattern="\\.mapping_metrics.csv$"), sep = "/")
+if ( file.exists(dragen_mapping_metrics) ) {
```

Comment:
Just to be safe I'd also add `& length(dragen_mapping_metrics) == 1`

Reply:
updated in https://github.com/umccr/RNAsum/pull/65/commits/318d01fbe5dfe002604be9b830789b9e0233f558